test/test_delete.py

```diff
@@ -0,0 +1,76 @@
+import pytest
+
+from .base import TestBase
+from constants import TRACK_URI_AKPH_MIVEL_JATSZOL, TRACK_URI_NKS_FOLD, TRACK_URI_QUEEN_UNDER_PRESSURE, PLAYLIST_ID_QUEEN_LIVE
```

Comment:
Please format this import part: separate the imported items with newline between brackets